resources/aws/errors.go

```diff
@@ -64,24 +52,76 @@ func IsRouteNotFoundError(err error) bool {
 	return errgo.Cause(err) == routeNotFoundError
 }
 
-func IsSecurityGroupFind(err error) bool {
-	return errgo.Cause(err) == securityGroupFindError
+var securityGroupNotFoundError = errgo.New("security group not found")
+
+// IsSecurityGroupNotFound asserts securityGroupNotFoundError.
+func IsSecurityGroupNotFound(err error) bool {
+	return errgo.Cause(err) == securityGroupNotFoundError
 }
 
-func IsSubnetFind(err error) bool {
-	return errgo.Cause(err) == subnetFindError
+var subnetNotFoundError = errgo.New("subnet not found")
+
+// IsSubnetNotFound asserts subnetNotFoundError.
+func IsSubnetNotFound(err error) bool {
+	return errgo.Cause(err) == subnetNotFoundError
+}
+
+var vpcNotFoundError = errgo.New("vpc not found")
+
+// IsVpcNotFound asserts vpcNotFoundError.
+func IsVpcNotFound(err error) bool {
+	return errgo.Cause(err) == vpcNotFoundError
 }
 
-func IsVpcFindError(err error) bool {
-	return errgo.Cause(err) == vpcFindError
+// Delete errors.
+
+var resourceDeleteError = errgo.New("Couldn't delete resource, it lacks the necessary data (ID)")
+
+// IsResourceDelete asserts resourceDeleteError.
+func IsResourceDelete(err error) bool {
```

Comment:
What is this error about? I feel like it should be `IsRecourceDeleted`, but then it could just be `IsNotFound`. Anyway, I cannot find any usage of this matcher or its error. It can maybe just be removed.